openlibrary/plugins/openlibrary/js/partials.js

```diff
@@ -0,0 +1,16 @@
+export function initPartials() {
+    jQuery(window).load(function () {
+        $.ajax({
+            url: '/partials',
+            type: 'GET',
+            data: {
+                workid: $('.RelatedWorksCarousel').attr('workId'),
+                _component: true
+            },
+            datatype: 'json',
+            success: function (response) {
+                $('.RelatedWorksCarousel').append(response[0]);
+            }
```

Comment:
After we add the carousels to the page, we then need to run the code which instantiates all carousels on the page :).... But we should be careful to **only** do this for the carousels we're adding.

```
$('.carousel--progressively-enhanced');
```